dev/com.ibm.ws.st.jee.ui/src/com/ibm/ws/st/jee/ui/internal/Messages.properties

```diff
@@ -38,6 +28,7 @@ sharedLibAPIVisibilityLabel=API visibility
 sharedLibAPIVisibilityAPI=A&PI
 sharedLibAPIVisibilityIBMAPI=IB&M API
 sharedLibAPIVisibilitySpec=&Spec
+sharedLibAPIVisibilityStable=&Stable
```

Comment:
The & designates the accelerator key and two items in the same list cannot have the same accelerator key.  Maybe put the & before the 'b' in Stable.